Kitodo/src/main/java/org/kitodo/production/helper/metadata/legacytypeimplementations/LegacyContentFileHelper.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.production.helper.metadata.legacytypeimplementations;
+
+import java.io.File;
+
+import org.kitodo.api.dataformat.mets.FLocatXmlElementAccessInterface;
+import org.kitodo.api.ugh.ContentFileInterface;
+import org.kitodo.production.services.ServiceManager;
+import org.kitodo.production.services.dataformat.MetsService;
+
+/**
+ * Connects a legacy content file to a media file. This is a soldering class to
+ * keep legacy code operational which is about to be removed. Do not use this
+ * class.
+ */
```

Comment:
remove-todo is missing here

Reply:
The class cannot be removed before all places in the code where it is used have been re-implemented.